src/plot/axis/axis-ticks.js

```diff
@@ -177,6 +177,15 @@ class AxisTicks extends React.Component {
     const ticks = values.map((v, i) => {
       const pos = scale(v);
       const text = tickFormatFn(v, i);
+      
+      const tick = React.isValidElement(text) && !['tspan','a'].includes(text.type)
```

Comment:
You've got quite a number of lint errors coming off this chunk of code. I think this could be cleaned up by making a boolean variable like,

```
const isNotText = React.isValidElement(text) && !['tspan','a'].includes(text.type);
const tick = isNotText ? text : (
  <text {...textProps}
    className="rv-xy-plot__axis__tick__text"
    style={{...style, ...style.text}}
    >
    {text}
  </text>)
```

Reply:
Thanks! I'm not really experienced with pull requests, so bear with me :).
I'll update the PR soon
Or, feel free to update it yourself.

Reply:
Take your time. It's important to learn how to be comfortable doing PRs +1 for learning